packages/core/src/scss/components/foundation/modal/_modal.scss

```diff
@@ -174,7 +174,7 @@
 // Specific styles for displaying cart summary on mobile
 body.has-activeModal {
     .modal-body {
-        overflow: scroll;
+        overflow: auto;
```

Comment:
[overflow](https://www.w3schools.com/cssref/pr_pos_overflow.php) has four properties.

> `visible` - Default. The overflow is not clipped. The content renders outside the element's box
> `hidden` - The overflow is clipped, and the rest of the content is hidden
> `scroll` - Scrollbars are added. User must scroll to see all content
> `auto` - Similar to scroll, but adds scrollbars only when necessary

`hidden` will cut the message text, `visible` will break the page layout, `scroll` looks redundant. 

`auto` is ideal, however, as the error modal always auto resizes itself, scroll bars should not be seen.